packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -813,4 +813,7 @@ FlexibleColumnLayout.define();
 
 export default FlexibleColumnLayout;
 
-export type { MEDIA };
+export type {
+	MEDIA,
+	FCLLayoutChangeEventDetail,
```

Comment:
```suggestion
	FlexibleColumnLayoutLayoutChangeEventDetail,
```
Although I don't like the `LayoutLayout` part in the middle of this super long type we should stick to the full component name convention